git_pile/config.py

```diff
@@ -77,6 +89,30 @@ def __init__(self):
             except:
                 warn(f"could not set {key}={value} from git config")
 
+    def normalize(self, root):
+        import sys
+
+        worktrees = git_worktree_list(root)
+
+        # usual situation: the call is from main worktree where the result branch
+        # is normally checkout - config is already loaded correctly and both root
+        # and root / config.dir are in the list of worktrees
+        pile_worktree = op.realpath(op.join(root, self.dir))
+        worktree = op.realpath(root)
+        if worktree in worktrees and pile_worktree in worktrees:
+            return root
+
+        # try the opposite: assume root is actually the pile checkout, reload config
+        pile_worktree = worktree
+        for w in worktrees:
+            pile_dir = git(f"-C {w} config --get pile.dir", check=False, stderr=nul_f).stdout.strip()
+            if pile_worktree == op.realpath(op.join(w, pile_dir)):
+                self._load_config_values(w)
+                return w
+
+        sys.exit(0)
```

Comment:
Shouldn't we exit with a non-zero status?

Reply:
ugh... another leftover